Sources/PrimerSDK/Classes/PCI/Tokenization View Models/FormsTokenizationViewModel/CardFormPaymentMethodTokenizationViewModel.swift

```diff
@@ -940,7 +982,11 @@ extension CardFormPaymentMethodTokenizationViewModel: UITextFieldDelegate {
 
         var countryResults: [CountryCode] = []
 
-        for country in countries where country.country.lowercased().folding(options: .diacriticInsensitive, locale: nil).contains(query.lowercased().folding(options: .diacriticInsensitive, locale: nil)) == true {
+        for country in countries where
+        country.country.lowercased().folding(options: .diacriticInsensitive,
+                                             locale: nil).contains(query.lowercased()
+            .folding(options: .diacriticInsensitive,
+                     locale: nil)) == true {
```

Comment:
Im not sure this is more readable than before. Its a tricky one as it is quite complex.

Perhaps we could extract the `.lowercased().folding(options: .diacriticInsensitive, locale: nil)` since it is repeated? It would clean it up a lot IMO.

eg

```swift
for country in countries where
            country
            .country
            .lowercasedAndFolded()
            .contains(
                query.lowercasedAndFolded()
            ) {
     countryResults.append(country)
 }
 ```
 ```swift
 private extension String {
    func lowercasedAndFolded() -> String {
        self
        .lowercased()
        .folding(
            options: .diacriticInsensitive,
            locale: nil)
    }
}
```

Reply:
To tackle only formatting without any refactor, I would suggest one of the following to group the two objects being compared in a more coherent way (It really is an awful piece of code to make pretty!):

```swift
for country in countries 
    where country.country.lowercased().folding(options: .diacriticInsensitive, locale: nil)
        .contains(query.lowercased().folding(options: .diacriticInsensitive, locale: nil)) 
    {
        countryResults.append(country)
    }
```

or

```swift
for country in countries 
        where country
            .country
            .lowercased()
            .folding(options: .diacriticInsensitive, locale: nil)
            .contains(
                query
                    .lowercased()
                    .folding(options: .diacriticInsensitive, locale: nil)
            ) {
            countryResults.append(country)
        }
```

FWIW I prefer the latter, I dont mind trading number of lines for this pipeline-style readability

Reply:
Great point! I extracted the code into a private extension, but the format you suggested will be applied to other cases similar to this one. Thanks!

Reply:
PS: I did not figure out correctly what was going on in this line of code...